db/compaction/compaction_job.cc

```diff
@@ -2476,6 +2476,21 @@ std::string CompactionJob::GetTableFileName(uint64_t file_number) {
                        file_number, compact_->compaction->output_path_id());
 }
 
+Env::IOPriority CompactionJob::GetRateLimiterPriority() {
+  if (versions_ && versions_->GetColumnFamilySet() &&
+      versions_->GetColumnFamilySet()->write_controller()) {
+    WriteController* write_controller =
+        versions_->GetColumnFamilySet()->write_controller();
+    if (write_controller->NeedsDelay() || write_controller->IsStopped()) {
+      return Env::IO_USER;
+    } else if (write_controller->NeedSpeedupCompaction()) {
+      return Env::IO_HIGH;
+    }
```

Comment:
Speedup is not necessarily a problem and already leads to increased compaction parallelism. I am not sure we want to additionally increase I/O priority as it might speed up too much :) (thereby slowing other things down - like memtable flush).

Reply:
Will change it.